janus_core/single_point.py

```diff
@@ -19,7 +17,7 @@ def __init__(
         system: str,
         architecture: str = "mace_mp",
         device: str = "cpu",
```

Comment:
The only other question I have is whether there is a list of valid entries in which case you can use e.g. `Literal["cpu", "gpu"]`, but am approving.

Reply:
shall be , cpu, cuda, and mps at the moment  if I missed it we shall open a bug about it.